src/coreclr/vm/comutilnative.h

```diff
@@ -218,9 +218,8 @@ extern "C" void QCALLTYPE Interlocked_MemoryBarrierProcessWide();
 class ValueTypeHelper {
 public:
     static FCDECL1(FC_BOOL_RET, CanCompareBits, Object* obj);
-    static FCDECL2(FC_BOOL_RET, FastEqualsCheck, Object* obj1, Object* obj2);
+    static FCDECL1(UINT32, GetNumInstanceFieldBytes, MethodTable* mt);
```

Comment:
Nit: It would be nice to put it into a separate type. I expect we are going to have more MethodTable methods like this over time.
```
class MethodTableNative {
public:
    static FCDECL1(UINT32, GetNumInstanceFieldBytes, MethodTable* mt);
}
```